### DISCUSSION TITLE
did Avalonia commits die about a year ago?

### DISCUSSION THREAD

**Discussion by busitech** [AvaloniaUI/Avalonia#17067] [General]
It seems evident by the charts that something drastic happened to the Avalonia project about one year ago.  Based on the charts, the rate of commits (work) seems to have almost stopped, and then stayed that way.  Is there a public explanation available, or is this data misleading?

<img width="912" alt="Avalonia Commits Died in August 2023" src="https://github.com/user-attachments/assets/b0f05589-b687-4800-81b6-a81ede28bb69">

**Comment by rabbitism** [comment]
Your observation is correct. From my perspective, Team and contributors skyrocketed the commits in 11.0-beta period and before. There were some fundamental changes happening over that time, like composition rendering integration, ItemsPresenter remake, textblock remake, control theme onboard, mobile and wasm. That's why commits are extremely high. Now stable version is stable enough, only binary compatible feature is acceptable in 11.x period.

**Reply by chatterzhao** [reply]
The documentation for version 11.x and its corresponding sub-versions does not appear to have been updated in a timely manner.

**Reply by thevortexcloud** [reply]
You can submit a PR to update the docs here: https://github.com/AvaloniaUI/avalonia-docs

There is also an issue to request docs to be added.

https://github.com/AvaloniaUI/avalonia-docs/issues/384

**Comment by ahopper** [comment]
~~My guess is all the good things being developed for Avalonia Accelerate may have something to do with this :)~~
I must stop guessing, see below :)

**Comment by MrJul** [comment]
Note that we switched to merging PRs using a single squashed commit in September 2023, so "commits over time" became mostly "PRs over time" at that moment.

Features such as #13970 (68 commits) which would have been a peak on the graph before, are now indistinguishable from typo fixes.
